modules/alloy/src/main/ts/ephox/alloy/api/ui/InlineView.ts

```diff
@@ -246,9 +242,6 @@ const InlineView: InlineViewSketcher = Sketcher.single<InlineViewSpec, InlineVie
     showAt: (apis, component, anchor, thing) => {
       apis.showAt(component, anchor, thing);
     },
-    showWithin: (apis, component, anchor, thing, boxElement) => {
-      apis.showWithin(component, anchor, thing, boxElement);
-    },
```

Comment:
It was refactored in Phase I to allow the calling code to decide the bounds within which it should be positioned.

Reply:
ah. I'm still tempted to go back and read this phase-by-phase, rather than all at once, I thought it might take me longer to do that but having the context along the way might make it easier to follow.

Reply:
Yeah, I often refer to the things being changed in the separated PRs when required. It's smaller and more straightforward to follow than having it in a huge PR.